parser.mk

```diff
@@ -14,7 +14,12 @@ clean: parser-clean tests-clean
 .PHONY: parser-clean
 parser-clean:
 	find . -name \*.loT -o -name \*.out | xargs rm -f
-	find ./parser -name zephir.c -o -name zephir.h | xargs rm -f
+	find ./parser \
+		   -name zephir.c  \
+		-o -name zephir.h  \
+		-o -name scanner.c \
+		-o -name parser.c  | xargs rm -f
```

Comment:
In general I would like leave scanner and parse under version control

Reply:
Is there a reason why you want to keep them in the repo? They are always regenerated by the build scripts.

I would even suggest moving the generated files `scanner.c` and `parser.c` to a build directory so they don't get mixed up with the actual source code.

Reply:
Never mind, we can remove them.